_data/fpkinotifications.yml

```
@@ -27,6 +27,37 @@
 # ee_cdp_uri:
 # ee_ocsp_uri:  
 
+- notice_date: February 21, 2023
+  change_type: Intent to Issue CA Certificate 
+  system: US Treasury Root CA 
+  change_description: The SSA CA will undergo a key update and a new CA certificate will be issued from the US Treasury Root CA on 03/04/2023.  The new CA key will begin to be used on 04/01/2023 to sign new end-entity certificates.
+  contact: pki_ops@fiscal.treasury.gov
```

Treasury requested that the contact email be obscured to reduce harvesting.
```suggestion
  contact: pki_ops at fiscal.treasury dot gov
```